### ISSUE TITLE
[EPIC]: Improve accessibility and performance

### ISSUE THREAD

**Issue by irisvw** [fdnd-agency/nebulaxplorer#272]
## Description
- Accessibility and performance should be looked at and improved wherever possible. 
- We should aim to adhere to the WCAG AA standards (https://www.w3.org/TR/WCAG22/)
- Our website should conform to the A11Y checklist. The checklist is based on the WCAG standards, so conforming to the checklist should be enough to also clear WCAG AA. (https://www.a11yproject.com/checklist/)
- We should aim for a high performance and accessibility score on Lighthouse, and resolve all issues that can reasonably be resolved.
